package.json

```diff
@@ -1,59 +1,61 @@
-{
-  "name": "wowerlay",
-  "description": "Wowerlay is a popover library for Vue 3 applications.",
-  "version": "0.7.0",
-  "main": "./dist/wowerlay.umd.js",
-  "module": "./dist/wowerlay.es.js",
-  "types": "./dist/types/lib.d.ts",
-  "repository": {
-    "url": "git+https://github.com/teamseodo/wowerlay.git",
-    "type": "git"
-  },
-  "bugs": {
-    "url": "https://github.com/teamseodo/wowerlay/issues"
-  },
-  "homepage": "https://github.com/teamseodo/wowerlay#readme",
-  "keywords": [
-    "vue 3",
-    "wowerlay",
-    "popover"
-  ],
-  "license": "MIT",
-  "files": [
-    "/dist",
-    "README.md"
-  ],
-  "scripts": {
-    "dev": "cross-env MODE=demo vite",
-    "build": "ts-node -T --project tsconfig-build.json scripts/build.ts",
-    "build:demo": "cross-env MODE=demo vite build",
-    "script:build": "tsc --project tsconfig-build.json && cross-env MODE=production vite build",
-    "watch": "cross-env MODE=production vite build --watch"
-  },
-  "devDependencies": {
-    "@types/fs-extra": "^9.0.13",
-    "@types/node": "^16.10.3",
-    "@types/prompts": "^2.0.14",
-    "@vitejs/plugin-vue-jsx": "^1.2.0",
-    "bottom-sheet-vue3": "^1.0.2",
-    "chalk": "^4.1.2",
-    "cross-env": "^7.0.3",
-    "eslint": "8.7.0",
-    "eslint-config-airbnb-typescript-prettier": "5.0.0",
-    "execa": "^5.1.1",
-    "fast-glob": "^3.2.7",
-    "fs-extra": "^10.0.0",
-    "highlight.js": "^11.3.1",
-    "prettier": "2.5.1",
-    "prompts": "^2.4.2",
-    "sass": "^1.42.1",
-    "ts-node": "^10.2.1",
-    "typescript": "4.5.2",
-    "vite": "^2.6.4",
-    "vite-plugin-dts": "^0.8.3",
-    "vue": "^3.2.0"
-  },
-  "peerDependencies": {
-    "vue": "^3.2.0"
-  }
-}
+{
+  "name": "wowerlay",
+  "description": "Wowerlay is a popover library for Vue 3 applications.",
+  "version": "0.8.0",
```

Comment:
Only version changed here, for no reason git is showing like whole file is changed.